<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimWin</class>
 <widget class="QMainWindow" name="SimWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="bench.qrc">
    <normaloff>:/bench.svg</normaloff>:/bench.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnSim">
        <property name="text">
         <string>Add Analysis</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnRun">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStop">
        <property name="text">
         <string>Abort</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPlot">
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="status">
        <property name="text">
         <string>&lt;Status&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="FileFrame" name="fileFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>515</width>
         <height>392</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="beforeFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Before/After each Analysis</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnHideba">
               <property name="text">
                <string>-</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Before:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="beforeText"/>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>After</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="afterText"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="topArea" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>529</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Simulation">
    <property name="title">
     <string>&amp;Simulation</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionAbort"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="actionEditNet"/>
    <addaction name="actionPlot"/>
    <addaction name="actionLogWindow"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Simulation"/>
   <addaction name="menu_Window"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>&amp;Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAbort">
   <property name="text">
    <string>Abort</string>
   </property>
  </action>
  <action name="actionPlot">
   <property name="text">
    <string>&amp;Plot</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="bench.qrc">
     <normaloff>:/bench.svg</normaloff>:/bench.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>aboutQt</string>
   </property>
  </action>
  <action name="actionLogWindow">
   <property name="text">
    <string>&amp;Log Window</string>
   </property>
  </action>
  <action name="actionEditNet">
   <property name="text">
    <string>&amp;Edit Net/Schem.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileFrame</class>
   <extends>QFrame</extends>
   <header>.fileframe</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="bench.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnHideba</sender>
   <signal>clicked(bool)</signal>
   <receiver>widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>SimWin</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRun</sender>
   <signal>clicked()</signal>
   <receiver>actionRun</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStop</sender>
   <signal>clicked()</signal>
   <receiver>actionAbort</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPlot</sender>
   <signal>clicked()</signal>
   <receiver>actionPlot</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
